relay-quotas/src/redis.rs

```diff
@@ -176,6 +180,7 @@ impl RedisRateLimiter {
         quotas: &[Quota],
         item_scoping: ItemScoping<'_>,
         quantity: usize,
+        over_accept_once: bool,
```

Comment:
nit: using a boolean makes reading the code calling this function harder to read, could we use an enum instead? An example is available in the `processor.rs` file in this diff. Example:

```rust
enum Overaccept {
    Once,
    No,
}
```

Reply:
I get your point, but you could apply this reasoning to any boolean function arg. I'd rather keep it a bool and rely on IDEs to annotate the arguments you're passing.

Reply:
this is totally my doing.  and yes, i'd do this to any boolean arg :).

Reply:
Agreed with @jjbayer , this is overkill in here.  If we had 3 and more values we want to accept that we could use enum for that, right now it's only 1 function with one marker flag.

Reply:
one day you too will despise boolean args.  but i'm not going to die on this hill today :)